mm2src/rpc/lp_commands.rs

```diff
@@ -128,7 +129,13 @@ pub async fn enable(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>, String>
         mature_confirmations: coin.mature_confirmations(),
     };
     let res = try_s!(json::to_vec(&res));
-    Ok(try_s!(Response::builder().body(res)))
+    let res = try_s!(Response::builder().body(res));
+
+    if let MmCoinEnum::UtxoCoin(_) = &coin {
```

Comment:
This should be also done for QtumCoin, Qrc20Coin, ZCoin, Bch, SlpToken as they are all a subset of UTXO coins. And only in native mode (UtxoRpcClientEnum::Native).

Reply:
In Electrum mode, our nodes can overload the remote light wallet mode servers because they will all start broadcasting attempts to them.